src/util/tests/envVariablesValidations.test.ts

```diff
@@ -17,7 +17,7 @@
 import dotenv from 'dotenv';
 
 import { checkEnvVariable } from '../envVariablesValidations';
-import { checkExpectedVariables, checkUnexpectedVariables } from '../envVariablesValidations';
```

Comment:
If you remove the function import and it's literally the only place where that function is used, please remove the function as well (but note I don't think it should be removed, just altered to cater for optional vars.)